Source/WTF/wtf/Vector.h

```diff
@@ -496,6 +496,7 @@ class VectorBuffer : private VectorBufferBase<T, Malloc> {
     bool allocateBuffer(size_t newCapacity)
     {
         // FIXME: This should ASSERT(!m_buffer) to catch misuse/leaks. https://bugs.webkit.org/show_bug.cgi?id=250801
+        ASSERT(!m_buffer);
```

Comment:
Please remove the comment now that you fixed it.  (The comment serves no purpose after adding the assertion.)

Reply:
Actually, this is hitting a lot of failures on [mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/4006), so those need to be investigated first.

Reply:
Abigail and I took a look at the code in `Vector.h`, and we found that one of the following two conditions must be true where the `ASSERT()` was added:

1. `m_buffer` is `nullptr`, or
2. `m_buffer` is set to `inlineBuffer()`.

Because condition #2 wasn't being checked in the original patch, that caused crashes on Debug builds.